crates/runtime/src/http/v1/nsql.rs

```diff
@@ -82,7 +82,7 @@ pub(crate) async fn post(
         match df.get_arrow_schema(t).await {
             Ok(schm) => {
                 let c = CreateTableBuilder::new(Arc::new(schm), format!("public.{t}").as_str());
-                table_create_stms.push(c.build_postgres());
+                table_create_stms.push(c.build_sqlite());
```

Comment:
Is it broken with postgres now?

Reply:
It requires the `postgres` feature to be enabled since that function is now feature gated - which doesn't make sense to require that the nsql code takes a dependency on that feature.